client/src/components/userProfiles/UserHistory.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+
+const UserHistory = () => {
+  const [userWins, setUserWins] = useState();
```

Comment:
initialize state with a value of the same datatype that you expect the variable to hold

Reply:
oops!